pkg/helper/roles/helpers.go

```diff
@@ -23,6 +23,19 @@ const (
 	RosaUpgradeAccRolesModeAuto = "ROSAUpgradeAccountRolesModeAuto"
 )
 
+// GeOperatorRolePrefixFromClusterName returns a valid operator role prefix from the cluster name
```

Comment:
@machi1990 Thanks for the GoDoc. We definitely need more of this 👍

Reply:
👍 Yes.